src/yul/parser/statement/function_definition.rs

```diff
@@ -112,9 +112,7 @@ where
         // special low-level "near" call instructions, able to set up their own
         // exception handlers.
         // sanitize_function
-        if let Some(error) = P::sanitize_function(&identifier, &mut arguments, location, lexer) {
-            return Err(error);
-        }
+        let _ = P::sanitize_function(&identifier, &mut arguments, location, lexer)?;
```

Comment:
No need for `let _` if the return result is `()`.

Reply:
Did you run `cargo clippy`?

Reply:
fixed, thank you!